.github/workflows/pythonpackage.yml

```diff
@@ -53,9 +53,7 @@ jobs:
       - name: Test with pytest
         run: |
           pip install pytest
-          # RECIRQ_CHESS_TEST_SEED: make quantum_chess tests determnistic
-          # RECIRQ_IMPORT_FAILSAFE: skip tests on unsupported Cirq configurations
-          RECIRQ_CHESS_TEST_SEED=789 RECIRQ_IMPORT_FAILSAFE=y pytest -v
```

Comment:
this is a call to `pytest` (don't be confused by the preceding environment variable declarations). We cannot have this removed. It tests the whole repository.